### PR TITLE
apply basic local file tests

### PR DESCRIPTION
**Pull request by hainv-ohio**

### CODE REVIEW

File: src/Reveal.Sdk.Dom.Tests/Data/DataSources/ExcelDataSourceFixture.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿using Reveal.Sdk.Dom.Core.Constants;
+using Reveal.Sdk.Dom.Data;
```
Comment (vulh-infragistics) [inline src/Reveal.Sdk.Dom.Tests/Data/DataSources/ExcelDataSourceFixture.cs:2]: Remove unused `using`

File: src/Reveal.Sdk.Dom.Tests/Data/DataSources/ExcelDataSourceFixture.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSources
+{
+    public class ExcelDataSourceFixture
+    {
+        [Fact]
+        public void Constructor_SetProviderToGoogleAnalytics4_WhenConstructed()
```
Comment (vulh-infragistics) [inline src/Reveal.Sdk.Dom.Tests/Data/DataSources/ExcelDataSourceFixture.cs:15]: Wrong name

File: src/Reveal.Sdk.Dom.Tests/Data/DataSources/JsonDataSourceFixture.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSources
+{
+    public class JsonDataSourceFixture
+    {
+        [Fact]
+        public void Constructor_SetProviderToGoogleAnalytics4_WhenConstructed()
```
Comment (vulh-infragistics) [inline src/Reveal.Sdk.Dom.Tests/Data/DataSources/JsonDataSourceFixture.cs:15]: Update unit test name

**Review by recca5p — APPROVED** [review]

File: src/Reveal.Sdk.Dom.Tests/Data/DataSources/ExcelDataSourceFixture.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
+﻿using Reveal.Sdk.Dom.Core.Constants;
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSources
+{
+    public class ExcelDataSourceFixture
+    {
+        [Fact]
+        public void Constructor_SetProviderToMicrosoftExcel_WhenConstructed()
```
Comment (vulh-infragistics) [inline src/Reveal.Sdk.Dom.Tests/Data/DataSources/ExcelDataSourceFixture.cs:10]:
```suggestion
        public void Constructor_SetProviderToMicrosoftExcel_WhenCalled()
```

File: src/Reveal.Sdk.Dom.Tests/Data/DataSources/JsonDataSourceFixture.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
+﻿using Reveal.Sdk.Dom.Core.Constants;
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSources
+{
+    public class JsonDataSourceFixture
+    {
+        [Fact]
+        public void Constructor_SetProviderToJSON_WhenConstructed()
```
Comment (vulh-infragistics) [inline src/Reveal.Sdk.Dom.Tests/Data/DataSources/JsonDataSourceFixture.cs:10]:
```suggestion
        public void Constructor_SetProviderToJSON_WhenCalled()
```

File: src/Reveal.Sdk.Dom.Tests/Data/DataSources/LocalFileDataSourceFixture.cs [hunk 6]
```diff
@@ -0,0 +1,20 @@
+﻿using Reveal.Sdk.Dom.Core.Constants;
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSources
+{
+    public class LocalFileDataSourceFixture
+    {
+        [Fact]
+        public void Constructor_SetProviderToLocalFile_WhenConstructed()
```
Comment (vulh-infragistics) [inline src/Reveal.Sdk.Dom.Tests/Data/DataSources/LocalFileDataSourceFixture.cs:10]:
```suggestion
        public void Constructor_SetProviderToLocalFile_WhenCalled()
```

File: src/Reveal.Sdk.Dom.Tests/Data/DataSources/CsvDataSourceFixture.cs [hunk 7]
```diff
@@ -0,0 +1,20 @@
+﻿using Reveal.Sdk.Dom.Core.Constants;
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSources
+{
+    public class CsvDataSourceFixture
+    {
+        [Fact]
+        public void Constructor_SetProviderToCSV_WhenConstructed()
```
Comment (vulh-infragistics) [inline src/Reveal.Sdk.Dom.Tests/Data/DataSources/CsvDataSourceFixture.cs:10]:
```suggestion
        public void Constructor_SetProviderToCSV_WhenCalled()
```

**Review by vulh-infragistics — APPROVED** [review]

**Review by vulh-infragistics — APPROVED** [review]

**Pull request merged**